JSTutorial/hw_8/task_4/Team.js

```diff
@@ -0,0 +1,103 @@
+class Team {
+    #name
+    #sprint_duration
+    #release_date
+    #daily_standup_time
+    #teammates = []
+    #tasks = []
+
+    constructor(name, sprint_duration, release_date, daily_standup_time, teammates, tasks) {
+        this.name = name
+        this.sprint_duration = sprint_duration
+        this.release_date = release_date;
+        this.daily_standup_time = daily_standup_time;
+        this.teammates = teammates;
+        this.tasks = tasks;
+    }
+
+    get name() {
+        return this.#name;
+    }
+
+    set name(name) {
+        if (typeof name !== "string") {
+            throw new Error("Invalid name type");
+        }
+        this.#name = name;
+    }
+
+    get sprint_duration() {
+        return this.#sprint_duration;
+    }
+
+    set sprint_duration(duration) {
+        if (typeof duration !== "number") {
+            throw new Error("Invalid duration type");
+        }
+        this.#sprint_duration = duration;
+    }
+
+    get release_date() {
+        return this.#release_date;
+    }
+
+    get daily_standup_time() {
+        return this.#daily_standup_time;
+    }
+
+    set daily_standup_time(duration) {
+        if (typeof duration !== "number") {
+            throw new Error("Invalid duration type");
+        }
+        return this.#daily_standup_time = duration;
+    }
+
+    getNumberOfTeammates() {
+        return this.#teammates.length;
+    }
+
+    addTeammate(newTeammate) {
+        if (!(newTeammate instanceof ITSpecialist)) {
+            throw new Error("Invalid teammate");
+        }
+        this.#teammates.push(newTeammate);
+    }
+
+    removeTeammate({name}) {
+        if (typeof name !== "string") {
+            throw new Error("Invalid name type");
+        }
+        this.#teammates.splice(this.#teammates.findIndex(specialist => specialist.name() === name), 1);
```

Comment:
Do not use brackets for getters / setters
https://javascript.info/property-accessors#getters-and-setters